src/articulation.ts

```diff
@@ -296,7 +296,7 @@ export class Articulation extends Modifier {
 
     const initialOffset = getInitialOffset(note, position);
 
-    const padding = this.musicFont.lookupMetric(`articulation.${glyph.getCode()}.padding`, 0);
+    const padding = Tables.currentMusicFont().lookupMetric(`articulation.${glyph.getCode()}.padding`, 0);
```

Comment:
We now use `Tables.currentMusicFont()` across VexFlow to get the first font in the music font stack. In the big Font PR, the Element class no longer has a `this.musicFont` instance variable. That approach caused issues when some elements had a music font that was different than the ones on the global font stack.

Reply:
No change required.